ChartsDemo-macOS/PlaygroundChart.playground/Pages/RadarChart.xcplaygroundpage/Contents.swift

```diff
@@ -73,32 +73,32 @@ for _ in 0..<cnt
 }
 //: ### RadarChartDataSet
 let set1 = RadarChartDataSet(values: entries1, label: "Last Week")
-set1.colors = [NSUIColor(red: CGFloat(103 / 255.0), green: CGFloat(110 / 255.0), blue: CGFloat(129 / 255.0), alpha: 1.0)]
-set1.fillColor = NSUIColor(red: CGFloat(103 / 255.0), green: CGFloat(110 / 255.0), blue: CGFloat(129 / 255.0), alpha: 1.0)
+set1.colors = [NSColor(red: CGFloat(103 / 255.0), green: CGFloat(110 / 255.0), blue: CGFloat(129 / 255.0), alpha: 1.0)]
```

Comment:
There's no need to wrap the division in a `CGFloat` initializer.

Reply:
Fixed